specification/dartLangSpec.tex

```diff
@@ -673,26 +674,104 @@ \subsection{Scoping}
 \LMLabel{scoping}
 
 \LMHash{}%
-A \Index{namespace} is a mapping of names denoting declarations to actual declarations.
-Let $NS$ be a namespace.
-We say that a name $n$ \Index{is in} $NS$ if $n$ is a key of $NS$.
-We say a declaration $d$ \NoIndex{is in} $NS$ if a key of $NS$ maps to $d$.
+A \Index{namespace} is a partial function that maps names to namespace values.
+A \Index{name} is a lexical token which is an \synt{IDENTIFIER},
+an \synt{IDENTIFIER} followed by \lit{=}, or
+an \synt{operator},
+or \code{unary-};
+and a \Index{namespace value} is
+a declaration, a namespace, or the special value \ConflictValue{}
+(\ref{conflictMergingOfNamespaces}).
+
+\LMHash{}%
+The graph of a namespace \NamespaceName{} is the set
```

Comment:
True, we're only using 'graph' in this sense in this single location, and it's only used to allow us to talk about keys, values, and bindings (`n -> V`), so that's a nice simplification! Done.